lstchain/io/tests/test_eventselection.py

```diff
@@ -0,0 +1,21 @@
+import pytest
+from lstchain.io import EventSelector
+
+def test_event_selection(simulated_dl1_file):
+   from lstchain.io.io import dl1_params_lstcam_key
+   import pandas as pd
+
+   data = pd.read_hdf(simulated_dl1_file, key=dl1_params_lstcam_key)
```

Comment:
I would propose testing no with some complicated example data but to create a custom table with some hand-picked values so you can make sure the correct events are filtered away.

Reply:
Ok